test/mocha/core.js

```diff
@@ -75,6 +75,25 @@ describe('core MochaWrapper semantics', function () {
 		});
 	});
 
+  /**
+   * Temporarily replace describe so that we can assert on the passed params.
```

Comment:
Nice. I'd probably have done this part in the tape tests instead, but this works fine